float32 radius          # stretch radius
float32 height          # height of the widest stretch
time time_stamp         # time stamp